source/user-manual/agent/agent-management/remote-upgrading/wpk-list.rst

```diff
@@ -8,32 +8,35 @@ WPK List
 Linux
 -----
 
-.. |WPK_Linux| replace:: `wazuh_agent_v|WAZUH_CURRENT|_linux_x86_64.wpk <https://packages.wazuh.com/4.x/wpk/linux/x86_64/wazuh_agent_v|WAZUH_CURRENT|_linux_x86_64.wpk>`__ (`sha512 <https://packages.wazuh.com/|WAZUH_CURRENT_MAJOR|/checksums/wazuh/|WAZUH_CURRENT|/wazuh_agent_v|WAZUH_CURRENT|_linux_x86_64.wpk.sha512>`__)
+.. |WPK_Linux_DEB| replace:: `wazuh_agent_v|WAZUH_CURRENT|_linux_amd64.deb.wpk <https://packages.wazuh.com/4.x/wpk/linux/deb/amd64/wazuh_agent_v|WAZUH_CURRENT|_linux_amd64.deb.wpk>`__ (`sha512 <https://packages.wazuh.com/|WAZUH_CURRENT_MAJOR|/checksums/wazuh/|WAZUH_CURRENT|/wazuh_agent_v|WAZUH_CURRENT|_linux_amd64.deb.wpk.sha512>`__)
+.. |WPK_Linux_RPM| replace:: `wazuh_agent_v|WAZUH_CURRENT|_linux_x86_64.rpm.wpk <https://packages.wazuh.com/4.x/wpk/linux/rpm/x86_64/wazuh_agent_v|WAZUH_CURRENT|_linux_x86_64.rpm.wpk>`__ (`sha512 <https://packages.wazuh.com/|WAZUH_CURRENT_MAJOR|/checksums/wazuh/|WAZUH_CURRENT|/wazuh_agent_v|WAZUH_CURRENT|_linux_x86_64.rpm.wpk.sha512>`__)
 
-+--------------+------------------+--------------+-------------------------------+
-| Distribution | Version          | Architecture | WPK Package                   |
-+==============+==================+==============+===============================+
-|    Linux     | |WAZUH_CURRENT|            |    64bit     | |WPK_Linux|                   |
-+--------------+------------------+--------------+-------------------------------+
++--------------+-----------------+--------------+-------------------------------+
+| Distribution | Version         | Architecture | WPK Package                   |
```

Comment:
The tables in this file look a bit odd, maybe we could re format them since we are here.